src/app/views/App.tsx

```diff
@@ -84,10 +87,27 @@ class App extends Component<IAppProps, IAppState> {
     });
   }
 
+  private collectDeviceCharacteristicsTelemetry() {
+    const deviceProperties = {
+      deviceHeight: screen.height,
+      deviceWidth: screen.width,
+      deviceScreenSize: getDeviceScreenSize(screen.width),
```

Comment:
This needs to be browserSize, device stats are interesting, browser stats are the ones that impact how we render the page